src/components/cc-zone-input/cc-zone-input.js

```diff
@@ -317,16 +317,16 @@ export class CcZoneInput extends withResizeObserver(LitElement) {
           border: 0;
           box-sizing: border-box;
           display: block;
-          margin: -0.5rem;
+          margin: -0.5em;
```

Comment:
Decided not to change it because to get back to a round 8px size, the `em` would get quite complex (more than 3 decimals).
As long as it's does not cause any visual issue, I decided to keep simple values.